### PR TITLE
finish week 4

### PR DESCRIPTION
**Pull request by nvergel**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -82,11 +82,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         Message userMessage = new Message(userId, userText.get(), timeCreated);
         ofy().save().entity(userMessage).now();
 
-      } else {// Maybe throw error
-        System.out.print("No message found");
+      } else {
+        throw new IllegalArgumentException("No message found");
       }
     } else {// Maybe throw error
```
Comment (gradywardgoogle) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:88]: nit: plz remove this comment.

File: portfolio/src/main/java/com/google/sps/servlets/YourBootstrapper.java [hunk 2]
```diff
@@ -4,6 +4,7 @@
 import javax.servlet.ServletContextEvent;
 import com.googlecode.objectify.ObjectifyService;
 
+// listener executed by web.xml, required for objectify
```
Comment (gradywardgoogle) [inline portfolio/src/main/java/com/google/sps/servlets/YourBootstrapper.java:7]: nice! Thanks

Comment (gradywardgoogle) [reply portfolio/src/main/java/com/google/sps/servlets/YourBootstrapper.java:7]: Maybe name this class something more descriptive then? Like "ObjectifyServletBootstrapper"?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (nvergel) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:88]: oh, oops

File: portfolio/src/main/java/com/google/sps/servlets/YourBootstrapper.java (hunk 2 above)
Comment (nvergel) [reply portfolio/src/main/java/com/google/sps/servlets/YourBootstrapper.java:7]: done

**Pull request merged**